src/content/en/2019/javascript.md

```diff
@@ -0,0 +1,338 @@
+downloaded.
+
+-  Preloading a resource with `<link rel="preload"> `tells the browser to download this resource
+   as soon as possible. This is especially helpful for critical resources which are discovered late
+   in the page loading process (e.g., javascript located at the bottom of your HTML) and are
+   otherwise downloaded last.
+-  Using `<link rel="prefetch">`  tells the browser to take advantage of any idle time it has to
+   fetch these resources needed for future navigations
+
+**So, how many sites use preload and prefetch directives?**
+
+<insert graphic of metric 01_14>
+
+For all sites measured in HTTP Archive, **14.33%** of desktop sites and **14.84%** of mobile sites
+use `<link rel="preload"> `for scripts on their page.
+
+For prefetch:
+
+<insert graphic of metric 01_16>
+
+For both mobile and desktop, **0.08% (~0.1%)** of pages leverage prefetch for any of their
+scripts.
+
+# Newer APIs
+
+JavaScript continues to evolve as a language. A new version of the language standard itself, known
+as ECMAScript, is released every year with new APIs and features passing proposal stages to become a
+part of the language itself.
+
+With HTTP Archive, we can take a look at any newer API that is supported (or is about to be) and see
+how widespread its usage is. These APIs may already be used in browsers that support them _or_ with
+an accompanying polyfill to make sure they still work for every user.
+
+**How many sites use the following APIs?**
+
+-  [Atomics](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics)
+-  [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl)
+-  [Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)
+-  [SharedArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
+-  [WeakMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap)
+-  [WeakSet](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet)
+
+<insert graphic of metric 01_18>
+
+Atomics (**0.38%**) and SharedArrayBuffer (**0.2%**) are barely visible on this chart since they are
+used on such few pages. Each of the other APIs here 
+
+It is important to note that the numbers here are approximations and they do not leverage
+[UseCounter](https://chromium.googlesource.com/chromium/src.git/+/master/docs/use_counter_wiki.md)
+to measure feature usage.
+
+# Source maps
+
+In many build systems, JavaScript files undergo minification to minimize its size and transpilation
+for newer language features that are not yet supported in many browsers. Moreover, language
+supersets like [TypeScript](https://www.typescriptlang.org/) compile to an output that can look
+extremely different from the original source code. For all these reasons, the final code served to
```

Comment:
I don't think `extremely` is correct in this context if we use TypeScript as an example of the source language - compiling down to pretty much the same what you write is one of its main selling points and design goals. I'd suggest either using a different source language or changing the language (no pun intended) here a bit.

Reply:
```suggestion
noticeably different from the original source code. For all these reasons, the final code served to
```